var/spack/repos/builtin/packages/gcc-runtime/package.py

```diff
@@ -44,9 +47,15 @@ class GccRuntime(Package):
         "ubsan",
     ]
 
+    # libgfortran ABI
+    provides("fortran-rt", "gfortran")
```

Comment:
drop the latter?

Reply:
So, the idea is that `fortran-rt` will prevent using two different runtimes in the same unification set, while `gfortran` will encode soname compatibility. Does that make sense to you?